Thu, 16 Jan 2020 07:12:23 - INFO - Connect to DB
Thu, 16 Jan 2020 07:12:23 - INFO - TRIZMA DB import
Thu, 16 Jan 2020 07:12:23 - INFO - 123TV DB import
Thu, 16 Jan 2020 07:13:00 - INFO - Connect to DB
Thu, 16 Jan 2020 07:13:00 - INFO - TRIZMA DB import
Thu, 16 Jan 2020 07:13:00 - INFO - 123TV DB import
Thu, 16 Jan 2020 07:13:21 - INFO - Connect to DB
Thu, 16 Jan 2020 07:13:21 - INFO - TRIZMA DB import
Thu, 16 Jan 2020 07:13:21 - INFO - 123TV DB import
Thu, 16 Jan 2020 07:14:07 - INFO - Connect to DB
Thu, 16 Jan 2020 07:14:08 - INFO - TRIZMA DB import
Thu, 16 Jan 2020 07:14:08 - INFO - 123TV DB import
Thu, 16 Jan 2020 07:14:30 - INFO - Connect to DB
Thu, 16 Jan 2020 07:14:30 - INFO - TRIZMA DB import
Thu, 16 Jan 2020 07:14:30 - INFO - 123TV DB import
Thu, 16 Jan 2020 07:16:20 - INFO - Connect to DB
Thu, 16 Jan 2020 07:16:20 - INFO - TRIZMA DB import
Thu, 16 Jan 2020 07:16:20 - INFO - 123TV DB import
Thu, 16 Jan 2020 07:16:26 - INFO - Connect to DB
Thu, 16 Jan 2020 07:16:26 - INFO - TRIZMA DB import
Thu, 16 Jan 2020 07:16:26 - INFO - 123TV DB import
Thu, 16 Jan 2020 07:20:08 - INFO - Connect to DB
Thu, 16 Jan 2020 07:20:08 - INFO - TRIZMA DB import
Thu, 16 Jan 2020 07:20:08 - INFO - 123TV DB import
Thu, 16 Jan 2020 07:24:38 - INFO - Connect to DB
Thu, 16 Jan 2020 07:24:38 - INFO - TRIZMA DB import
Thu, 16 Jan 2020 07:24:38 - INFO - 123TV DB import
Thu, 16 Jan 2020 07:25:53 - INFO - Connect to DB
Thu, 16 Jan 2020 07:25:53 - INFO - TRIZMA DB import
Thu, 16 Jan 2020 07:25:53 - INFO - 123TV DB import
Thu, 16 Jan 2020 07:26:43 - INFO - Connect to DB
Thu, 16 Jan 2020 07:26:43 - INFO - TRIZMA DB import
Thu, 16 Jan 2020 07:26:43 - INFO - Failed on TRIZMA DB import at: 2020-01-16_07-11-36, for following calling party: 49304237062, for agent id: trz_madamovic, with error:(pyodbc.DataError) ('22001', '[22001] [Microsoft][ODBC Driver 17 for SQL Server][SQL Server]String or binary data would be truncated. (8152) (SQLExecDirectW)')
[SQL: insert into records values('CC2Servicenummer','Belgrad','Nicht übermittelt','trz_madamovic',                                 '49304237062','','Nicht übermittelt','49304237062_Wednesday, January 15, 2020_12:27:36 PM','MIL_ADAMOV')]
(Background on this error at: http://sqlalche.me/e/9h9h)
Thu, 16 Jan 2020 07:26:43 - INFO - 123TV DB import
Thu, 16 Jan 2020 07:27:27 - INFO - Connect to DB
Thu, 16 Jan 2020 07:27:27 - INFO - TRIZMA DB import
Thu, 16 Jan 2020 07:27:27 - INFO - 123TV DB import
Thu, 16 Jan 2020 07:27:52 - INFO - Connect to DB
Thu, 16 Jan 2020 07:27:52 - INFO - TRIZMA DB import
Thu, 16 Jan 2020 07:27:52 - INFO - Failed on TRIZMA DB import at: 2020-01-16_07-11-36, for following calling party: 4968069879266, for agent id: trz_divankovic, with error:(pyodbc.DataError) ('22001', '[22001] [Microsoft][ODBC Driver 17 for SQL Server][SQL Server]String or binary data would be truncated. (8152) (SQLExecDirectW)')
[SQL: insert into records values('CC2Servicenummer','Belgrad','Nicht übermittelt','trz_divankovic',                                 '4968069879266','','Nicht übermittelt','4968069879266_Thursday, January 16, 2020_8:27:50 AM','DUS_IVANKO')]
(Background on this error at: http://sqlalche.me/e/9h9h)
Thu, 16 Jan 2020 07:27:52 - INFO - 123TV DB import
Thu, 16 Jan 2020 07:28:05 - INFO - Connect to DB
Thu, 16 Jan 2020 07:28:05 - INFO - TRIZMA DB import
Thu, 16 Jan 2020 07:28:05 - INFO - 123TV DB import
Thu, 16 Jan 2020 07:28:33 - INFO - Connect to DB
Thu, 16 Jan 2020 07:28:33 - INFO - TRIZMA DB import
Thu, 16 Jan 2020 07:28:33 - INFO - Failed on TRIZMA DB import at: 2020-01-16_07-11-36, for following calling party: 493853921080, for agent id: trz_madamovic, with error:(pyodbc.DataError) ('22001', '[22001] [Microsoft][ODBC Driver 17 for SQL Server][SQL Server]String or binary data would be truncated. (8152) (SQLExecDirectW)')
[SQL: insert into records values('CC2Servicenummer','Belgrad','Nicht übermittelt','trz_madamovic',                                 '493853921080','','Nicht übermittelt','493853921080_Wednesday, January 15, 2020_2:57:56 PM','MIL_ADAMOV')]
(Background on this error at: http://sqlalche.me/e/9h9h)
Thu, 16 Jan 2020 07:28:33 - INFO - 123TV DB import
Thu, 16 Jan 2020 07:32:32 - INFO - Connect to DB
Thu, 16 Jan 2020 07:32:32 - INFO - TRIZMA DB import
Thu, 16 Jan 2020 07:32:33 - INFO - 123TV DB import
Thu, 16 Jan 2020 07:32:54 - INFO - Connect to DB
Thu, 16 Jan 2020 07:32:54 - INFO - TRIZMA DB import
Thu, 16 Jan 2020 07:32:54 - INFO - 123TV DB import
Thu, 16 Jan 2020 07:34:04 - INFO - Connect to DB
Thu, 16 Jan 2020 07:34:04 - INFO - TRIZMA DB import
Thu, 16 Jan 2020 07:34:04 - INFO - Failed on TRIZMA DB import at: 2020-01-16_07-11-36, for following calling party: 4917630771978, for agent id: trz_madamovic, with error:(pyodbc.DataError) ('22001', '[22001] [Microsoft][ODBC Driver 17 for SQL Server][SQL Server]String or binary data would be truncated. (8152) (SQLExecDirectW)')
[SQL: insert into records values('CC2Servicenummer','Belgrad','Nicht übermittelt','trz_madamovic',                                 '4917630771978','','Nicht übermittelt','4917630771978_Wednesday, January 15, 2020_4:04:02 PM','MIL_ADAMOV')]
(Background on this error at: http://sqlalche.me/e/9h9h)
Thu, 16 Jan 2020 07:34:04 - INFO - 123TV DB import
Thu, 16 Jan 2020 07:34:17 - INFO - Connect to DB
Thu, 16 Jan 2020 07:34:17 - INFO - TRIZMA DB import
Thu, 16 Jan 2020 07:34:17 - INFO - 123TV DB import
Thu, 16 Jan 2020 07:34:25 - INFO - Connect to DB
Thu, 16 Jan 2020 07:34:25 - INFO - TRIZMA DB import
Thu, 16 Jan 2020 07:34:25 - INFO - 123TV DB import
Thu, 16 Jan 2020 07:34:48 - INFO - Connect to DB
Thu, 16 Jan 2020 07:34:48 - INFO - TRIZMA DB import
Thu, 16 Jan 2020 07:34:48 - INFO - Failed on TRIZMA DB import at: 2020-01-16_07-11-36, for following calling party: 4964197191699, for agent id: trz_madamovic, with error:(pyodbc.DataError) ('22001', '[22001] [Microsoft][ODBC Driver 17 for SQL Server][SQL Server]String or binary data would be truncated. (8152) (SQLExecDirectW)')
[SQL: insert into records values('CC2Servicenummer','Belgrad','Nicht übermittelt','trz_madamovic',                                 '4964197191699','','Nicht übermittelt','4964197191699_Wednesday, January 15, 2020_6:22:22 PM','MIL_ADAMOV')]
(Background on this error at: http://sqlalche.me/e/9h9h)
Thu, 16 Jan 2020 07:34:48 - INFO - 123TV DB import
Thu, 16 Jan 2020 07:35:30 - INFO - Connect to DB
Thu, 16 Jan 2020 07:35:30 - INFO - TRIZMA DB import
Thu, 16 Jan 2020 07:35:30 - INFO - 123TV DB import
Thu, 16 Jan 2020 07:35:45 - INFO - Connect to DB
Thu, 16 Jan 2020 07:35:45 - INFO - TRIZMA DB import
Thu, 16 Jan 2020 07:35:45 - INFO - 123TV DB import
Thu, 16 Jan 2020 07:38:18 - INFO - Connect to DB
Thu, 16 Jan 2020 07:38:18 - INFO - TRIZMA DB import
Thu, 16 Jan 2020 07:38:18 - INFO - 123TV DB import
Thu, 16 Jan 2020 07:39:38 - INFO - Connect to DB
Thu, 16 Jan 2020 07:39:38 - INFO - TRIZMA DB import
Thu, 16 Jan 2020 07:39:38 - INFO - 123TV DB import
Thu, 16 Jan 2020 07:41:39 - INFO - Connect to DB
Thu, 16 Jan 2020 07:41:39 - INFO - TRIZMA DB import
Thu, 16 Jan 2020 07:41:39 - INFO - 123TV DB import
Thu, 16 Jan 2020 07:43:15 - INFO - Connect to DB
Thu, 16 Jan 2020 07:43:15 - INFO - TRIZMA DB import
Thu, 16 Jan 2020 07:43:15 - INFO - 123TV DB import
